torchmetrics/functional/text/bert.py

```diff
@@ -478,10 +481,10 @@ def bert_score(
     This implemenation follows the original implementation from `BERT_score`_
 
     Args:
-        predictions:
+        preds:
             Either an iterable of predicted sentences or a `Dict[str, torch.Tensor]` containing `input_ids` and
             `attention_mask` `torch.Tensor`.
-        references:
+        target:
```

Comment:
let's also include the old names with a just warning that they are deprecated in favor of...